app/javascript/dashboard/components/ui/Banner.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="banner" :class="bannerClasses">
+    <span>
+      {{ bannerMessage }}
+      <a :href="hrefLink" rel="noopener noreferrer nofollow" target="_blank">
+        {{ hrefLinkText }}
+      </a>
+    </span>
+    <woot-button
+      v-if="isActionButton"
+      size="small"
+      variant="link"
+      icon="arrow-right"
+      color-scheme="primary"
+      class-names="banner-action__button"
+      @click="onClick"
+    >
+      {{ actionButtonLabel }}
+    </woot-button>
+    <woot-button
+      v-if="isCloseButton"
+      size="small"
+      variant="link"
+      color-scheme="warning"
+      icon="dismiss-circle"
+      class-names="banner-action__button"
+      @click="onClickClose"
+    >
+    </woot-button>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    bannerMessage: {
+      type: String,
+      default: '',
+    },
+    hrefLink: {
+      type: String,
+      default: '',
+    },
+    hrefLinkText: {
+      type: String,
+      default: '',
+    },
+    isActionButton: {
+      type: Boolean,
+      default: false,
+    },
+    actionButtonLabel: {
+      type: String,
+      default: '',
+    },
+    bgColorScheme: {
+      type: String,
+      default: '',
+    },
+    isCloseButton: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    bannerClasses() {
+      return [this.bgColorScheme];
+    },
+  },
+  methods: {
+    onClick(e) {
+      this.$emit('click', e);
+    },
+    onClickClose(e) {
+      this.$emit('close', e);
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.banner {
```

Comment:
Check if there are other occurrences of .banner styles after moving. If it's a big effort we will track in a separate issue.